D4e/d4e_functions.py

```diff
@@ -68,16 +68,22 @@ async def attack(
         roll = roll_list[1]
 
     # Roll the dice
-    roll_string: str = f"{roll}{ParseModifiers(attack_modifier)}"
+    # print(roll)
+    try:
+        roll_string: str = f"{roll}{ParseModifiers(attack_modifier)}"
+    except:
+        roll_string = roll
     dice_result = d20.roll(roll_string)
+    # print(dice_result)
 
     async_session = sessionmaker(engine, expire_on_commit=False, class_=AsyncSession)
-    Tracker = await get_tracker(ctx, engine)
-    Condition = await get_condition(ctx, engine)
+    guild = await initiative.get_guild(ctx, None)
+    Tracker = await get_tracker(ctx, engine, id=guild.id)
+    Condition = await get_condition(ctx, engine, id=guild.id)
 
     try:
         async with async_session() as session:
-            result = await session.execute(select(Tracker).where(Tracker.name == target))
+            result = await session.execute(select(Tracker.id).where(Tracker.name == target))
```

Comment:
Good catch! Similar idea, we might want to look over how Avrae is storing/loading the character/monster details into class objects so we can centralize all the database related work into singular places.

Reply:
Taken to Issue: #105